spec.md

```diff
@@ -22,67 +22,78 @@ bytes (e.g. `32`), a number in bits (e.g. `7` bit), a choice of lengths (e.g.
 
-What Tox Does:
+Achieved by using ephemeral keys (TODO: how are they used in the current
+protocol? is the problem actually solved?)
 
--   Authentication: user's address is its public key, thus "adding friend"
-    actually means verifying key (false in case of toxme?); drawback is that
-    after being compromised you have to generate absolutely new identity; also
-    it complicates working with multiple devices; (TODO: how about some web of
-    trust, master keys and subkeys here?)
+#### Reliability
 
--   End-to-end encryption: all the messages are encrypted via keys derived via
-    DH, thus keys are only known to sender and receiver and are never
-    transfered over network
+-   Tox is supposed to be fully distributed network which doesn't depend on any
+    Single Point of Failure. This is achieved by using DHT and direct (P2P)
+    connections between the peers in the network. Entry point is still
+    required. Process of connecting to the network is called `bootstrapping`.
+    Bootstrapping can and should be done using multiple entry points.
 
--   Forward secrecy (?): can be achieved by using ephemeral keys (TODO: how are
-    they used in the current protocol? is the problem actually solved?)
+-   Tox is supposed to work under (almost) any kind of NAT and firewall; this
+    is achieved by using hole-punching, UPnP and TCP-relays.
 
--   Reliability:
+-   Resistance to basic DoS and other poisoning.
 
-    -   Tox is supposed to be fully decentralized network which doesn't depend
-        on any Single Point of Failure; this is achieved by using DHT, though
-        you still need an entry point;
+#### Zero-conf (easy-to-use)
 
-    -   Tox is supposed to work under (almost) any kind of NAT and firewall;
-        this is achieved by using hole-punching, UPnP and TCP-relays;
+End-user should be able to use the messenger that ***Just Works***. Security
+that is too hard to use is useless.
 
-    -   Resistance to basic DoS and other poisoning.
+### What Tox Does NOT
 
--   (Near-)Zero-conf: end-user should be able to *just* use the messenger;
+#### Anonymity
 
-What Tox Does NOT:
+Unless TCP mode is used, participants communicate directly to each other. One
+of the reasons for this is that relaying real-time video is rather too costly
+(in terms of load) and also means delays.
 
--   Tox does not care about anonymity; Unless TCP mode is used, participants
-    communicate directly to each other; One of the reasons for this is that
-    relaying real-time video is rather too costly (in terms of load) and also
-    means delays;
+Your IP Address is exposed to nodes in your friendlist. They can link your ID
+directly to IP Address.
 
-    -   Your IP Address is exposed to nodes in your friendlist; They can link
-        your ID directly to IP Address;
+Temporary DHT nodes and onion tunnels are used to find friends, so that your ID
+cannot be linked to your IP based solely on publicly available data (TODO: i.e.
```

Comment:
Replace i.e. with e.g. 
I.e. is for putting things in other words. E.g. is for examples.